ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java

```diff
@@ -27,6 +27,7 @@
 import io.confluent.ksql.rest.server.KsqlRestConfig;
 import io.confluent.ksql.rest.server.state.ServerState;
 import io.confluent.ksql.security.KsqlSecurityExtension;
+import io.confluent.ksql.util.FileWatcher;
```

Comment:
Is this used?

Reply:
Yes. This line was added because I moved the package that FileWatcher was previously in.